Formula/lib/libdpp.rb

```diff
@@ -0,0 +1,27 @@
+class Libdpp < Formula
+  desc "C++ Discord API Bot Library"
+  homepage "https://github.com/brainboxdotcc/DPP"
+  url "https://github.com/brainboxdotcc/DPP/releases/download/v10.0.28/DPP-10.0.28.tar.gz"
+  sha256 "aa0c16a1583f649f28ec7739c941e9f2bf9c891c0b87ef8278420618f8bacd46"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "libsodium"
+  depends_on "openssl@3"
+  depends_on "opus"
+  depends_on "pkg-config"
```

Comment:
Is this needed at runtime?

Reply:
No, but it's required to build for voice support (hence libsodium and libopus), otherwise voice support is disabled and the user would need to reinstall the package once pkg-config is installed.

Reply:
That sounds like it's only needed on the build

Reply:
> That sounds like it's only needed on the build

Yeah, but without pkg-config, users won't be able to use voice support at runtime! We would like users to be able to use voice support out of the box.

Reply:
I'm confused, is the detection at runtime or build time?

Reply:
> I'm confused, is the detection at runtime or build time?

Detection is at build time. If pkg-config is not found then it doesn't define `HAVE_VOICE` which means users can not use voice support until they rebuild D++. If you want to look at this, you can view the detection [here](https://github.com/brainboxdotcc/DPP/blob/master/library/CMakeLists.txt#L129).

Reply:
So that means pkg-config can be removed as a runtime dependency right?

Reply:
> So that means pkg-config can be removed as a runtime dependency right?

Again, doing that means users manually have to install pkg-config to use voice support. Meaning, they have to reinstall D++ as it's a built time configuration.

We should just let this formula download pkg-config so voice support works instantly.

Reply:
But it never builds on a user system in any supported configuration, or are you saying that Homebrew builds it, but if it doesn't find pkg-config on runtime it loses the ability to support voice untill Homebrew rebuilds it?

Reply:
Yes I am saying that 😃, sorry I've been wording it oddly!

Reply:
So the detection _is_ at runtime then, not at compile time

Reply:
I guess? I'm a bit confused now 😅

As long as we can keep this so D++ can use Voice Support!